src/acl/external/time_quota/ext_time_quota_acl.cc

```diff
@@ -54,14 +50,14 @@ const char *program_name;
 
 TDB_CONTEXT *db = nullptr;
 
-#define KEY_LAST_ACTIVITY            "last-activity"
-#define KEY_PERIOD_START             "period-start"
-#define KEY_PERIOD_LENGTH_CONFIGURED "period-length-configured"
-#define KEY_TIME_BUDGET_LEFT         "time-budget-left"
-#define KEY_TIME_BUDGET_CONFIGURED   "time-budget-configured"
+const char* KeyLastActivity = "last-activity";
+const char* KeyPeriodStart = "period-start";
+const char* KeyPeriodLengthConfigured = "period-length-configured";
+const char* KeyTimeBudgetLeft = "time-budget-left";
+const char* KeyTimeBudgetConfigured = "time-budget-configured";
```

Comment:
If you want to remove macros, please replace them with constants. Current PR code replaces them with modifiable variables.

```suggestion
const auto KeyLastActivity = "last-activity";
const auto KeyPeriodStart = "period-start";
const auto KeyPeriodLengthConfigured = "period-length-configured";
const auto KeyTimeBudgetLeft = "time-budget-left";
const auto KeyTimeBudgetConfigured = "time-budget-configured";
```